test/Utilities/Microsoft.Testing.TestInfrastructure/TestAssetFixtureBase.cs

```diff
@@ -63,6 +63,8 @@ protected virtual void Dispose(bool disposing)
 
             _disposedValue = true;
         }
+
+        _nugetGlobalPackagesDirectory.Dispose();
```

Comment:
We create a singleton of `NuGetGlobalPackagesFolder` and we dispose in assembly cleanup. See `AcceptanceFixture.cs`